@here/harp-mapview/lib/PickHandler.ts

```diff
@@ -101,18 +102,22 @@ export interface PickResult {
  * Handles the picking of scene geometry and roads.
  */
 export class PickHandler {
+    readonly rayCaster: PickingRaycaster;
     private readonly m_plane = new THREE.Plane(new THREE.Vector3(0, 0, 1));
     private readonly m_roadPicker?: RoadPicker;
 
     constructor(
         readonly mapView: MapView,
         readonly camera: THREE.Camera,
+        width: number,
```

Comment:
Do we recreate the PickHandler when we resize?

Reply:
No, we don't, I already saw we have a bug there